File: src/main/java/gr/apt/repository/RestHolidaysRepository.java
package gr.apt.repository;

import gr.apt.persistence.holiday.RestHolidays;
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;

import javax.enterprise.context.ApplicationScoped;
import java.math.BigInteger;

@ApplicationScoped
public class RestHolidaysRepository implements PanacheRepositoryBase<RestHolidays, BigInteger> {
}

File: src/main/java/gr/apt/repository/RestHolidaysRepository.kt
package gr.apt.repository

import gr.apt.persistence.holiday.RestHolidays
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase
import java.math.BigInteger
import javax.enterprise.context.ApplicationScoped

@ApplicationScoped
class RestHolidaysRepository : PanacheRepositoryBase<RestHolidays?, BigInteger?>